docs/src/documentation/03-components/02-structure/card/03-accessibility.mdx

```diff
@@ -0,0 +1,139 @@
+
+- Expandable sections can be toggled with `Enter` or `Space` when focused
+- Focus is managed automatically when expanding/collapsing sections
+- When multiple expandable sections exist in a Card, users can navigate between them using `Tab`
+
+### Screen reader experience
+
+#### Card component
+
+Screen readers will announce:
+
+- The Card title as specified by the `titleAs` prop (when provided)
+- The close button with the custom label specified in `labelClose` prop
+- The section identifier provided in `dataA11ySection` (when provided)
+
+#### CardSection component
+
+Screen readers will announce:
+
+- The CardSection title as specified by the `titleAs` prop
+- The expanded/collapsed state for expandable sections
+- The content within the section when expanded
+
+### Important restrictions
+
+- **CardSection with actions and expandability**: When using the `actions` prop with an expandable CardSection (`expandable={true}` or with `onClick` handler), prop `actions` should be undefined or passed a non-interactive element. This prevents focus management issues and keyboard navigation confusion. Interactive elements in this context can create conflicts with the expansion functionality, leading to poor accessibility.
+
+Example of what to avoid:
+
+```jsx
+<CardSection
+  expandable
+  title="Flight details"
+  actions={<Button>Edit</Button>} // Avoid interactive elements here
+>
+  {/* content */}
+</CardSection>
+```
+
+Recommended approach:
+
+```jsx
+<CardSection
+  expandable
+  title="Flight details"
+  actions={<Badge type="info">Economy</Badge>} // Non-interactive element is acceptable
+>
+  {/* content */}
+</CardSection>
+```
+
+### Best practices
+
+#### Card best practices
+
+- Use semantic heading levels with the `titleAs` prop to maintain proper document structure
+- Provide a descriptive `labelClose` that clearly explains the action (e.g., "Close flight details")
+- When creating complex card layouts and using SkipNavigation component on the page, you can use the `dataA11ySection` prop connect the Card with this component
+
+#### CardSection best practices
+
+- Avoid placing interactive elements in the `actions` prop of expandable CardSection components
+- Maintain logical heading hierarchy between Card titles and CardSection titles
+
+### Examples with accessibility features
+
+#### Basic Card with accessibility props
+
+```jsx
+<Card
+  title="Flight details"
+  titleAs="h3"
+  dataA11ySection="flight-summary"
+  labelClose="Close flight details"
+  onClose={() => {}}
+>
+  <CardSection>
+    <Stack direction="column" spacing="small">
+      <Text>Prague to Barcelona</Text>
+      <Text type="secondary">June 21, 2023</Text>
+    </Stack>
+  </CardSection>
+</Card>
+```
+
+Screen readers will announce this as a section identified as `flight-summary` with a heading level 3 of "Flight details", followed by the content, with a close button labeled "Close flight details".
```

Comment:
Do screen readers announce heading levels? And what does "announce this as a section identified as `flight-summary`" mean exactly?

Reply:
yes, VO announces the level of the heading. the `flight-summary` in dataA11ySection was polished.
<img width="802" alt="image" src="https://github.com/user-attachments/assets/930791b1-3d63-4f49-a41b-559fdc28cb41" />